aexpect/ops_windows.py

```diff
@@ -503,6 +503,7 @@ def find_unused_port(session, start_port=10024):
     :returns: unused port that was found
     :rtype: int
     """
+    # pylint: disable=C0301
```

Comment:
Would be great to rebase this branch on top of the main branch once https://github.com/avocado-framework/aexpect/pull/126 is merged.